src/bun.js/bindings/bindings.cpp

```diff
@@ -2455,6 +2455,48 @@ JSC__JSValue JSC__JSValue__fromEntries(JSC__JSGlobalObject* globalObject, ZigStr
     return JSC::JSValue::encode(object);
 }
 
+JSC__JSValue JSC__JSValue__keys(JSC__JSGlobalObject* globalObject, JSC__JSValue objectValue) {
```

Comment:
I like this approach, but instead of a custom implementation of `Object.keys`, we should use the one available in JSC. You can use `JSC::ownPropertyKeys` and the contents of the function should look almost identical to https://github.com/oven-sh/WebKit/blob/b4de09f41b83e9e5c0e43ef414f1aee5968b6f7c/Source/JavaScriptCore/runtime/ObjectConstructor.cpp#L269, except the argument is passed in rather than coming from a callframe

Reply:
Nice this is better !